pkg/bbgo/session.go

```diff
@@ -40,6 +40,7 @@ type ExchangeSession struct {
 	Secret       string             `json:"secret,omitempty" yaml:"secret,omitempty"`
 	Passphrase   string             `json:"passphrase,omitempty" yaml:"passphrase,omitempty"`
 	SubAccount   string             `json:"subAccount,omitempty" yaml:"subAccount,omitempty"`
+	ConfigPath   string
```

Comment:
ditto, Can you pull out the config things to another PR? if you would like this PR to get merged first.

I think the config saving stuff needs to be designed carefully.

Reply:
done